app/src/main/java/org/oppia/app/topic/TopicFragmentPresenter.kt

```diff
@@ -4,20 +4,80 @@ import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
 import androidx.fragment.app.Fragment
+import androidx.fragment.app.FragmentManager
+import androidx.fragment.app.FragmentPagerAdapter
+import androidx.viewpager.widget.ViewPager
+import com.google.android.material.tabs.TabLayout
+
+import org.oppia.app.R
 import org.oppia.app.databinding.TopicFragmentBinding
 import org.oppia.app.fragment.FragmentScope
+import org.oppia.app.topic.overview.TopicOverviewFragment
+import org.oppia.app.topic.play.TopicPlayFragment
+import org.oppia.app.topic.review.TopicReviewFragment
+import org.oppia.app.topic.train.TopicTrainFragment
 import javax.inject.Inject
 
 /** The controller for [TopicFragment]. */
 @FragmentScope
 class TopicFragmentPresenter @Inject constructor(
   private val fragment: Fragment
 ) {
+  private lateinit var tabLayout: TabLayout
+  private lateinit var viewPager: ViewPager
+  private val tabIcons =
+    intArrayOf(
+      R.drawable.ic_overview_white_24dp,
+      R.drawable.ic_overview_white_24dp,
```

Comment:
Shouldn't we be using the other icons for these, instead?

Reply:
Ya i did't get icons so once i get them will replace this

Reply:
You can add TODO to replace these icons.

Reply:
Resolved